java/core/src/java/org/apache/orc/Reader.java

```diff
@@ -144,23 +160,27 @@
    * Get the file tail (footer + postscript)
```

Comment:
Could you add the tag to `getWriterVersion`? And please check the other functions once more.

Reply:
Oh. Thanks. Sure.